modules/retrieving-the-logs-with-the-kubectl-command.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * troubleshooting/retrieving-and-analyzing-the-collector-logs-and-pod-status.adoc
+:_module-type: PROCEDURE
+[id="retrieving-the-logs-with-the-kubectl-command_{context}"]
+= Retrieving the logs with the `kubectl` command
+
+Kubernetes is the easiest way for you to retrieve the logs, provided you have the necessary access to do so. 
```

Comment:
**suggestion:** 
- Kubernetes is not a process, so it cant be the easiest way. 
- `provided you have the necessary access to do so.` That's what Prerequisites section is for. 

Maybe: 
```
Kubernetes captures logs from each container in a running Pod. To fetch the logs, use the `kubectl logs` command.
```

Reply:
ack